django/settings/base.py

```diff
@@ -130,10 +130,14 @@ def get_env_bool_value(key: str, default: bool):
 # Auth backends
 # https://docs.djangoproject.com/en/3.1/topics/auth/customizing/#specifying-authentication-backends
 
-AUTHENTICATION_BACKENDS = [
-    "mozilla_django_oidc.auth.OIDCAuthenticationBackend",
-    "django.contrib.auth.backends.ModelBackend",
-]
+AUTHENTICATION_BACKENDS = (
+    []
+    if get_env_bool_value("DISABLE_AUTH", False)
```

Comment:
use ENV == "test" instead?

Reply:
I prefer being explicit about the `DISABLE_AUTH` env varibale and leave to the deployment the logic of setting this var depending on the env